### ISSUE TITLE
Why the script doesn't closes the browser?

### ISSUE THREAD

**Issue by StasToken** [facebook/php-webdriver#330]
Hi. I have a small script that simply logs in, tell me what could be the reason scrpit after work does not close the browser window, the browser remains open. How to fix it?

```
<?php
namespace Facebook\WebDriver;
use Facebook\WebDriver\Remote\DesiredCapabilities;
use Facebook\WebDriver\Remote\RemoteWebDriver;
require_once('vendor/autoload.php');
$host = 'http://localhost:4444/wd/hub';
$capabilities = DesiredCapabilities::chrome();
$driver = RemoteWebDriver::create($host, $capabilities, 50000,50000);
$driver->get("http://example.com/");
$driver->manage()->deleteAllCookies();
$driver->manage()->addCookie(array(
  'name' => 'cookie_name',
  'value' => 'cookie_value',
));
$cookies = $driver->manage()->getCookies();
$driver->quit();
?>
```

**Comment by AlcyZ** [comment]
I had the same problem a couple of months ago.
My problem was a bug in the chrome driver, that creates zombie processes. We solved the problem by using a different version of chrome driver.
Try to google "chrome driver session does not close". There are a lot of articles with the same issue.
Your code seems to be ok, i can't find any error there.

I hope i could help.

**Comment by OndraM** [comment]
@StasToken Hi, this exact script does work for me and closes the window properly (Chromium 53.0.2785.143 , ChromeDriver 2.22). I suggest the same as @AlcyZ wrote - trying different version of Chrome/ChromeDriver).
